stable/redis-ha/values.yaml

```diff
@@ -22,7 +22,7 @@ nodeSelector: {}
 tolerations: []
 
 ## Redis image version
-redis_image: quay.io/smile/redis:4.0.6r2
+redis_image: quay.io/smile/redis:latest
```

Comment:
I've pinged @viglesiasce to see if he can tag a `4.0.8r0` release for us

Reply:
@prydonius tag created.

Reply:
Thanks @smileisak. 
@prydonius values.yaml is tagged with quay.io/smile/redis:4.0.8r0